### ISSUE TITLE
Volar should display a better error message when a prop is missing from a custom component

### ISSUE THREAD

**Issue by rodrigocfd** [vuejs/language-tools#2092]
In the example below, I have a custom component which has a single, mandatory prop: `tam`.

If the prop is missing, Volar correctly indicates that something is wrong. However, when hovering the component, a *huge gibberish* is shown... as a developer, all I need to know is that `tam` prop is missing!

Here's a screenshot showing the issue:

![missing-prop](https://user-images.githubusercontent.com/1283868/199822670-feeadd90-8467-45ee-815b-28d7547e1e24.png)

Please consider a more developer-friendly error message, so we can find what's wrong immediately.

**Comment by johnsoncodehk** [comment]
Since v1.1 we support missing props hint, you can set `"vue.features.inlayHints.missingProps": true` to enable it.

**Comment by rodrigocfd** [comment]
OK, I enabled `"vue.features.inlayHints.missingProps": true`.

There is a missing `guitar` property in the `Guitar` component below. This is what I'm getting:

![screen](https://user-images.githubusercontent.com/1283868/233784061-42519abe-5c36-436c-8eb0-5a533dcd97ca.png)

Is this the intended message?

**Comment by johnsoncodehk** [comment]
No, what's your VSCode and Volar version?

**Comment by rodrigocfd** [comment]
VSCode v1.77.3
Volar v1.4.2

**Comment by johnsoncodehk** [comment]
Is `editor.inlayHints.enabled` enable in your VSCode settings?

**Comment by rodrigocfd** [comment]
It's configured as `"editor.inlayHints.enabled": "offUnlessPressed"`.

I just tried and it shows like this:

![missing-prop](https://user-images.githubusercontent.com/1283868/233848228-eb443088-30b2-4c71-ba29-210f5c06105a.png)

This behavior is correct, however the issue I opened is regarding the **mouse over** popup, which is just unintelligible stuff. I don't know if it could be further improved, but it would be really nice to have something actually useful on mouse over.

Seems like a missed opportunity.
